docker/start.sh

```diff
@@ -32,31 +32,48 @@ setup_cassandra_schema() {
     temporal-cassandra-tool --ep $CASSANDRA_SEEDS -k $VISIBILITY_KEYSPACE update-schema -d $VISIBILITY_SCHEMA_DIR
 }
 
+temporal_mysql_tool() {
+    echo "temporal-sql-tool --ep $MYSQL_SEEDS -u $MYSQL_USER --pw XXXX $MYSQL_CONNECT_ATTR $@"; 
+    { temporal-sql-tool --ep $MYSQL_SEEDS -u $MYSQL_USER --pw $MYSQL_PWD $MYSQL_CONNECT_ATTR $@; } 2> /dev/null
+}
+
+temporal_postgres_tool() {
+    echo "temporal-sql-tool --plugin postgres --ep $POSTGRES_SEEDS -u $POSTGRES_USER --pw XXXX -p $DB_PORT $@"; 
+    { temporal-sql-tool --plugin postgres --ep $POSTGRES_SEEDS -u $POSTGRES_USER --pw $POSTGRES_PWD -p $DB_PORT $@; } 2> /dev/null
+}
+
 setup_mysql_schema() {
     SCHEMA_DIR=$TEMPORAL_HOME/schema/mysql/v57/temporal/versioned
-
+    VISIBILITY_SCHEMA_DIR=$TEMPORAL_HOME/schema/mysql/v57/visibility/versioned
+    
     if [ "$MYSQL_TX_ISOLATION_COMPAT" == "true" ]; then
         MYSQL_CONNECT_ATTR='--connect-attributes tx_isolation=READ-COMMITTED'
     fi
-
```

Comment:
i think the password can be provided in an env var:
```
◉ 18:13:44 [markmark ~/src/temporalio-temporal] (master|m1u) $ ./temporal-sql-tool
...
   --password value, --pw value            password used for authentication when connecting to sql host [$SQL_PASSWORD]
...
```

so i wonder if it would be simpler to just set $SQL_PASSWORD (inside of `set +x`/`set -x`, to prevent it from being printed), and removing `--pw $MYSQL_PWD` from the command line.

Something along the lines of 
```bash
set +x
export SQL_PASSWORD=password123
set -x
temporal-sql-tool --ep $MYSQL_SEEDS -u $MYSQL_USER $MYSQL_CONNECT_ATTR create --db $DBNAME
```

(and, similarly, for `temporal-cassandra-tool`, it's `CASSANDRA_PASSWORD`)

Reply:
good idea. let me try this

Reply:
fixed. this is a far better solution as it still displays the happy path output of the command